NLogPerformance/Program.cs

```diff
@@ -51,13 +66,13 @@ static void Main(string[] args)
             var currentProcess = Process.GetCurrentProcess();
 
             GC.Collect(2, GCCollectionMode.Forced, true);
-            Thread.Sleep(2000); // Allow .NET runtime to do its background thing, before we start
+            System.Threading.Thread.Sleep(2000); // Allow .NET runtime to do its background thing, before we start
 
             Console.WriteLine("Executing performance test...");
             Console.WriteLine("");
-            Console.WriteLine("| Messages   | Size | Threads | Loggers |");
-            Console.WriteLine("|------------|------|---------|---------|");
-            Console.WriteLine("| {0,10} | {1,4} | {2,7} | {3,7} |", _messageCount, _messageSize, _threadCount, _loggerCount);
+            Console.WriteLine("| Logger Name      | Messages   | Size | Threads | Loggers |");
+            Console.WriteLine("|------------------|------------|------|---------|---------|");
+            Console.WriteLine("| {0,-16} | {1,10} | {2,4} | {3,7} | {4,7} |", _loggerName, _messageCount, _messageSize, _threadCount, _loggerCount);
```

Comment:
please use ` {1,10:N0} `

I don't like reading 1000000 while it could be 1.000.000.

I added them in https://github.com/NLog/NLog-Performance-tests/pull/10/files, but those are lost :(